website/public/js/services/taskServices.js

```diff
@@ -15,6 +15,30 @@
 
   function tasksFactory($rootScope, Shared, User) {
 
+    function addTask(listDef, fn) {
+      if (listDef.bulk) {
+        var tasks = listDef.newTask.split(/[\n\r]+/);
+        tasks.reverse();
+        _.each(tasks, function(task) {
+          fn(task);
+        });
+        listDef.bulk = false;
+      } else {
+        var task = listDef.newTask;
+        fn(task);
+      }
+      delete listDef.newTask;
+      delete listDef.focus;
+    }
+
+    function toggleBulk(list) {
+      if (typeof list.bulk === 'undefined') {
```

Comment:
I know you didn't write this part of the code, but this if is unnecessary.

`!undefined --> true` which is the same as `!false`